terraform/modules/happy-mesh-access-control/main.tf

```diff
@@ -34,15 +34,23 @@ resource "kubernetes_manifest" "linkerd_mesh_tls_authentication" {
       "labels"    = var.labels
     }
     "spec" = {
-      "identityRefs" = concat([for v in var.allow_mesh_services : {
-        "kind"      = "ServiceAccount"
-        "name"      = v.service_account_name != null && v.service_account_name != "" ? v.service_account_name : "${v.stack}-${v.service}-${var.deployment_stage}-${v.stack}"
-        "namespace" = var.k8s_namespace
-        }], local.allow_ingress_controller ? [{
-        "kind"      = "ServiceAccount"
-        "name"      = "nginx-ingress-ingress-nginx"
-        "namespace" = "nginx-encrypted-ingress"
-      }] : [])
+      "identityRefs" = concat(
+        [for v in var.allow_mesh_services : {
```

Comment:
would you mind pulling out these arrays as local variables so that its easier to read?